task/sql-tasks.js

```diff
@@ -67,7 +84,18 @@ async function task_1_3(db) {
  *
  */
 async function task_1_4(db) {
-    throw new Error("Not implemented");
+
+    let result = await db.query(`
+    SELECT 
+        CustomerID AS 'Customer Id',
+        COUNT(OrderID) AS 'Total number of Orders',
+        ROUND((COUNT(OrderID)/(SELECT COUNT(*) AS 'All' from Orders)) * 100, 5) AS '% of all orders'
+        FROM Orders
+    GROUP BY Orders.CustomerID

```

Comment:
if you're only 1 table is used in the query no need to write it in from of the column

Reply:
Sorry, I don't understand what is the problem. What do I need to do?